src/index.html

```diff
@@ -6,7 +6,316 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body id="home">
+    <div class="container">
+      <header class="header">
+        <a href="#home" class="header__logo-link">
+          <img
+            class="header__logo-img"
+            src="images/logo.png"
+            alt="logo"
+          >
+        </a>
+        <nav class="header__nav">
```

Comment:
it can be just `nav` 
```suggestion
        <nav class="nav">
```

Reply:
in BEM you can have block in block